ilastik/widgets/featureDlg.py

```diff
@@ -1,9 +1,7 @@
-from __future__ import print_function
```

Comment:
minor, but since we're on it: could we change the name of this file/class to `featureDialog/FeatureDialog`?

Reply:
It's mixed all over the place. maybe we could unify this some time, but if so, I would suggest to do that in another PR

Reply:
fair enough